styles/components/_remainingTime.scss

```diff
@@ -0,0 +1,13 @@
+.remaining-time {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__counter {
+    font-size: $font-size-primary;
+    font-weight: bold;
+    color: rgba(255, 0, 0, 0.8);
+    text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25),
+      4px 4px 40px rgba(255, 0, 0, 0.9);
+  }
```

Comment:
Remember the order in CSS:
/* Positioning /
 / Display & Box Model /
  / Color /
  / Text /
  / Other */